Homework/Buoi4/lib/bloc/listfood/list_food_state.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:equatable/equatable.dart';
+
+import '../../model/meal.dart';
+
+abstract class ListFoodState extends Equatable {
```

Comment:
Có 2 giáo phái triển khai lớp state bro nên tìm hiểu thêm nha :
- subclass : mỗi state tương ứng với 1 class và mỗi subclass sẽ phải có cú pháp [TênBloc] + [Action] + [State]
   - Triển khai bằng cách mỗi state sẽ là 1 lớp con extends từ abtract class state
   - Ví dụ như của bro nhé 
- single class: 1 class và các state sẽ biểu thị thông qua trạng thái thuộc sính status. 
   - Triển khai bằng các lưu thông tin state thông qua môt enum class 
   - Ví dụ
  `enum ListFoodStatus { initial, loading, success, failure }`
`class ListFoodState {`
`  ListFoodState({this.status = ListFood.initial});`
`   final ListFoodStatus status;`
`  ///final List<Data>  ...`
`  /// sử dụng hàm copyWith(..) để switch state`
`}`